osbuild/meta.py

```diff
@@ -697,17 +698,23 @@ def list_modules_for_class(self, klass: str) -> List[str]:
         if not module_path:
             raise ValueError(f"Unsupported nodule class: {klass}")
 
-        path = os.path.join(self.path, module_path)
-        modules = filter(lambda f: os.path.isfile(f"{path}/{f}") and not path.endswith(".meta.json"),
-                         os.listdir(path))
+        modules = []
+        for path in self.libdir.dirs:
+            path = os.path.join(path, module_path)
+            modules += [
+                f for f in os.listdir(path)
+                if os.path.isfile(f"{path}/{f}") and not path.endswith(".meta.json")
```

Comment:
Let's use `os.path.join` to do path concatenation.

Reply:
Sorry that this took so long, https://github.com/osbuild/osbuild/pull/1903 now contains some cleanups in prep for this PR.